traffic_ops/testing/api/v1/servers_test.go

```diff
@@ -71,19 +70,18 @@ func UpdateTestServers(t *testing.T) {
 	var alert tc.Alerts
 	alert, _, err = TOSession.UpdateServerByID(remoteServer.ID, remoteServer)
 	if err != nil {
-		t.Errorf("cannot UPDATE Server by hostname: %v - %v", err, alert)
+		t.Fatalf("cannot UPDATE Server by ID: %v - %v", err, alert)
```

Comment:
Good fix on the typo, but why is that fatal? The one below definitely should be.